src/test/java/org/apache/commons/io/input/BrokenInputStreamTest.java

```diff
@@ -34,7 +35,7 @@
  */
 public class BrokenInputStreamTest {
 
-    static final class CustomeException extends Exception {
+    static final class CustomException extends Exception {
```

Comment:
Good catch. There is no need to duplicate exception classes, just extract this class into its own file and reuse in all tests.

Reply:
Done.